english/craving/craving.md

```diff
@@ -0,0 +1 @@
+You are craving marshmellows covered in chocolate! You can't believe how powerful the craving is! You must have them now! You quickly put some in a bowl and cover them with Chocolate. [You begin eating!](eating-walls/eating-marshmellows.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.